cachalot/tests/read.py

```diff
@@ -862,6 +862,32 @@ def test_unmanaged_model(self):
         self.assert_tables(qs, UnmanagedModel)
         self.assert_query_cached(qs)
 
+    def test_per_query_do_not_cache_uncached_data(self):
+        """
+        Test that when a query is given the `cachalot_do_not_cache` attribute with True value,
+        the query will not be cached.
+        """
+        qs = Test.objects.all()
+        qs.query.cachalot_do_not_cache = True
+        with self.assertNumQueries(1):
+            data1 = list(qs.all())
+        with self.assertNumQueries(1):
+            # repeat run of the same query should trigger one more DB query
+            data2 = list(qs.all())
+        self.assertEqual(data1, data2)
```

Comment:
I can't possibly think of a reason why someone wouldn't want an evaluated queryset to be saved in memory... I think you wrote this because there was that capability right?

Reply:
Logically, it would make sense. But for those developing for a long time in Django, it'd feel weird. That's just my opinion. @beda42 @BertrandBordage